### PR TITLE
[feature] (image tool) support image load tool

### LINKED ISSUE #8981: [Feature] support image load tool to validate if the image is valid
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Description

After dumping image, should validate if the image is valid.

### Use case

```
./start_fe.sh --image image_path
```

if success, result will be
`Load image success. Image file image_path is valid` 

and if failed, result will be
`Load image failed. Image file image_path is invalid`
 

### Related issues

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Henry2SS**
# Proposed changes

Issue Number: close #8981 

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (Yes)
3. Has document been added or modified: (No Need)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — COMMENTED** [review]
Could you please add document for this feature:
`administrator-guide/operation/metadata-operation.md`

**Comment by Henry2SS** [comment]
> Could you please add document for this feature: `administrator-guide/operation/metadata-operation.md`

Done.

File: fe/fe-core/src/main/java/org/apache/doris/PaloFe.java [hunk 1]
```diff
@@ -303,6 +316,23 @@ private static void checkCommandLineOptions(CommandLineOptions cmdLineOpts) {
             } else {
                 System.exit(-1);
             }
+        } else if (cmdLineOpts.runImageTool()) {
+            File imageFile = new File(cmdLineOpts.getImagePath());
+            if (!imageFile.exists()) {
+                System.out.println("image does not exist: " + imageFile.getAbsolutePath() + " . Please put an absolute path instead");
+                System.exit(0);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/PaloFe.java:323]:
```suggestion
                System.exit(-1);
```

Comment (Henry2SS) [reply fe/fe-core/src/main/java/org/apache/doris/PaloFe.java]: Done.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**